pkg/envtest/envtest_test.go

```diff
@@ -349,4 +354,20 @@ var _ = Describe("Test", func() {
 			close(done)
 		}, 30)
 	})
+
+	Describe("Start", func() {
+		It("should raise an error", func(done Done) {
```

Comment:
Nit: Elaborate in the name why it should (not) raise an error, i.E. "should raise an error on invalid dir`